gems/sorbet-runtime/test/types/props/decorator.rb

```diff
@@ -173,11 +173,7 @@ class InnerStruct < T::Struct
   end
 
   describe 'validating prop values' do
-    it 'validates subdoc hashes have the correct values' do
-
-      assert_raises(TypeError) do
-        StructHash.new(the_hash: {'foo' => {}})
-      end
```

Comment:
Was this deleted just because it's identical to below? Is it worth keeping the `.new` call around and just deleting the `assert_raises`?